cg/meta/archive/archive.py

```diff
@@ -34,12 +44,29 @@ class SpringArchiveAPI:
     """Class handling the archiving of sample SPRING files to an off-premise location for long
     term storage."""
 
-    def __init__(self, housekeeper_api: HousekeeperAPI, status_db: Store):
+    def __init__(
+        self,
+        ddn_dataflow_client: DDNDataFlowClient,
+        housekeeper_api: HousekeeperAPI,
+        status_db: Store,
+    ):
         self.housekeeper_api: HousekeeperAPI = housekeeper_api
+        self.ddn_client: DDNDataFlowClient = ddn_dataflow_client
         self.status_db: Store = status_db
+        self.handler_map: Dict[str, ArchiveModels] = {
+            ArchiveLocationsInUse.KAROLINSKA_BUCKET: ArchiveModels(
+                file_model=MiriaFile.from_file_and_sample, handler=self.ddn_client
+            )
+        }
+
+    def call_corresponding_archiving_function(
```

Comment:
Yes, saw that this was weird when looking through this. I will address this in a separate PR wherein I let it return a Response object which we handle in the implementation. It could be that other implementations don't return a job_id is my guess.